stylesheets/components/Badge/Badges.css

```diff
@@ -0,0 +1,114 @@
+  /* Badge Container */
+  .badge__container {
+      position: relative;
+      width: max-content;
+  }
+/* Badge Types */
+  /* Default Badge */
+  .badge {
+      position: absolute;
+      top: -12px;
+      right: -12px;
+      font-size: 0.875rem;
+      font-weight: 500;
+      min-width: 24px;
+      min-height: 24px;
+      background-color: var(--primary-color);
+      color: var(--primary-text);
+      border-radius: 6px;
+  }
+
+  /* Badge on Icon */
+  .badge--icon {
+      top: -10px;
+      right: -10px;
+  }
+
+  /* Rounded Badge */
```

Comment:
you can remove such comments as the name of the class is descriptive enough.

Reply:
Thanks Sameer. Have removed the additional comments. 
Will keep this in mind from next time